lecture-loops.html

```diff
@@ -117,10 +119,20 @@
     <p>
       <b>Challenge Question:</b> Now that we have all the pieces, write a program that iterates through an array from the back to the front and prints out only the values at the odd indices.
     </p>
+
+    <pre><code>var myArray = [7, 24, 25, 9, 40, 41, 11, 60, 61];
+// your program should output: 60,41,9,24</code></pre>
+
+    <p>
+    <b>Challenge Question:</b> write a program that iterates through an array from the back to the front and prints out only the <b>odd</b> values.
```

Comment:
can we make one of these front-to-back and the other back-to-front for diversity's sake?